post-result.py.in

```diff
@@ -4,32 +4,215 @@ import os
 import subprocess
 import sys
 
```

Comment:
just curious why the filename suffix is `.in` rather than `.py`? Since the whole file is a python script.